### PR TITLE
Refactor ExecutiveDetails API to conform to new design

### PR DESCRIPTION
**Pull request by git-ashug**
Closes issue #65 

Code is getting compiled with passing unit test cases.

### CODE REVIEW

**Comment by git-ashug** [comment]
Hi @mayorJAY , @unicodeveloper , I have raised the PR, please verify and let me know if further changes are required

File: src/main/java/co/novu/api/executivedetails/ExecutiveDetailsApi.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import java.util.Map;
+
+import co.novu.api.executivedetails.responses.ExecutiveDetailsResponse;
+import retrofit2.Call;
+import retrofit2.http.GET;
+import retrofit2.http.QueryMap;
+
+public interface ExecutiveDetailsApi {
+	
+	@GET("execution-details")
```
Comment (mayorJAY) [inline src/main/java/co/novu/api/executivedetails/ExecutiveDetailsApi.java:12]: Please extract the endpoint into a variable

**Comment by git-ashug** [comment]
Hello @mayorJAY , I have incorporated you suggestion, please check now

**Review by mayorJAY — APPROVED** [review]
LGTM🚀

**Pull request merged**